ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -1137,7 +1137,7 @@ public void onCompletion(String result, Exception exception) {
       putFutures.add(future);
     }
     for (Future<String> future : putFutures) {
-      future.get(1, TimeUnit.SECONDS);
+      future.get(5, TimeUnit.SECONDS);
```

Comment:
Increased this as this takes longer than a second sometimes on my (slow) machine - even without this patch.